dev-lua/lanes/lanes-3.11.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="A lightweight, native, lazy evaluating multithreading library"
+DESCRIPTION="Lua Lanes is a Lua extension library providing \
+the possibility to run multiple Lua states in parallel. It is \
+intended to be used for optimizing performance on multicore CPU's \
+and to study ways to make Lua programs naturally parallel to begin with."
+HOMEPAGE="http://lualanes.github.io/lanes/"
+COPYRIGHT="2007-17 Asko Kauppi
+	Benoit Germain"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/LuaLanes/lanes/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="0a82fadeed96099b7bf90da92ece3862507a5e41a23778a7456b8adaf36e9180"
+PATCHES="lanes-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	lanes$secondaryArchSuffix = $portVersion
+	lib:core$secondaryArchSuffix
```

Comment:
This looks wrong. The library should be in $libDir/lua/5.3/lanes/, so no need to declare it.

Reply:
Ok, I'll try to look into it later :)